docs/Theming.md

```diff
@@ -0,0 +1,49 @@
+# Theming
+
+## Motivation
+We want to give Gardener landscape administrators the possibility to change the appearance of the Gardener Dashboard via configuration without the need to touch the code. Right now, we support basic theming, more theming options would be possible but are currently not planned.
+
+## Colors
+Gardener Dashboard as been built with Vuetify. We use Vuetify's built-in [theming support](https://vuetifyjs.com/en/features/theme/) to centrally configure colors that are used throughout the web application.
+Colors can be configured for both light and dark themes. Configuraion is done via the helm chart, see the respective theme section there. Colors cam be secified as HTM color code (e.g. #FF0000 for red) or by referncing a color from Vuetify's Material Design [Color Pack](https://vuetifyjs.com/en/styles/colors/#javascript-color-pack).
+
+The following colors can colors can be configured:
+
+| name                    | usage|
+| ----------------------- |------|
+| `primary`               | icons, chips, buttons, popovers, etc. |
+| `anchor`                | links |
+| `main-background`       | main navigation, login page   |
+| `main-navigation-title` | text color on main navigation |
+| `toolbar-background`    | background color for toolbars in cards, dialogs, etc. |
+| `toolbar-title`         | text color for toolbars in cards, dialogs, etc. |
+| `action-button`         | buttons in tables and cards, e.g. cluster details page |
+| `info`                  | Snotify info popups |
+| `warning`               | Snotify warning popups, warning texts |
+| `error`                 | Snotify error popups, error texts |
+
+## Logos and Icons
+It is also possible to exchange the Dashboard logo and icons. You can replace the assets folder via a volume mount in your deployment.
+
+The simplest solution is to create a configmap and add the assets to it:
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dashboard-assets
+data:
+  logo.svg: |
+    ...
+```
+
+then you can map the assets to your dashboard in your deployment like this:
+```
+...
+volumes:
+  - configMap:
+    items:
+    name: dashboard-assets
+  name: assets
+```
+
+Attention: You need to replace all assets in the `public/static/assets` folder, as mapping the volume will overwrite all files. It is not possible to exchange single files.
```

Comment:
we could integrate this even more in the charts, so that in the values.yaml file you only provide the icons that you want to replace and the helm chart takes care of the rest, like creating the `dashboard-assets` configmap and adapting the deployment. It would also make sure that the icons are defaulted, that were not overwritten in the values.yaml
Maybe an additional hacks script is required to achieve this defaulting of the configmap.

I guess it would help a lot so that landscape administrators do not need to patch the chart and possibly adapt it on the next dashboard release

Reply:
I like the idea. Do you think we should try to do this in this PR? Or move this to an issue?

Reply:
I guess we can ship this as is and improve in the future. So I'm also fine if we create an issue for it

Reply:
https://github.com/gardener/dashboard/issues/961